controls/SV-221588.rb

```diff
@@ -56,4 +56,22 @@
   tag legacy: ["SV-94635","V-79931"]
   tag cci: ["CCI-000169"]
   tag nist: ["AU-12 a"]
+
+  if input('siprnet_system') == true
+    impact 0.0
+    describe 'This Control is Not Applicable to systems on the SIPRNet.' do
```

Comment:
Again, no need to call out the DOD Network here ... This control is Not Applicable on Sensitive Networks